Objective-C/CBLDatabase.mm

```diff
@@ -53,6 +53,13 @@
 // How long to wait after a database opens before expiring docs
 #define kHousekeepingDelayAfterOpening 3.0
 
+// this variable defines the state of database
+typedef enum {
+    kCBLDatabaseStateClosed = 0,
+    kCBLDatabaseStateClosing,
+    kCBLDatabaseStateOpen,
```

Comment:
Should it be 'Opened'?

Reply:
I am okay with `Open` or `Opened`. will change to `Opened`, which will be in consistent with `Closed`?